dataclients/kubernetes/deploy/apply/routegroups_crd.yaml

```diff
@@ -1,128 +1,204 @@
-apiVersion: apiextensions.k8s.io/v1beta1
+
+---
+apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.4.0
+  creationTimestamp: null
   name: routegroups.zalando.org
 spec:
   group: zalando.org
-  version: v1
-  scope: Namespaced
   names:
-    kind: RouteGroup
-    singular: routegroup
-    plural: routegroups
     categories:
     - all
-    shortNames:
```

Comment:
No short name anymore?

Reply:
Bug in the CRD gen specification. Fix here: https://github.com/szuecs/routegroup-client/pull/8

Reply:
Updated the pr